### ISSUE TITLE
An error with AOFEX  api

### ISSUE THREAD

**Issue by zaneran9** [ccxt/ccxt#6922]
I execute this code, the transaction seems to have been established, but always returns an error, it needs to be repaired  urgently.

```python
import ccxt

symbols = 'BV/AQ'
amount = 1
price = 2.0
aofex = ccxt.aofex()
aofex.apiKey = 'xxxxx'
aofex.secret = 'xxxxx'

a = aofex.create_limit_buy_order(symbols, amount, price)
print(a)
```

```
Traceback (most recent call last):
  File "C:/Users/xxx/xxx/aofex/ccxt_aofex.py", line 10, in <module>
    a = aofex.create_limit_buy_order(symbols, amount, price)
  File "C:\Users\xxx\xxx\env\lib\site-packages\ccxt\base\exchange.py", line 1757, in create_limit_buy_order
    return self.create_order(symbol, 'limit', 'buy', *args)
  File "C:\Users\xxx\xxx\env\lib\site-packages\ccxt\aofex.py", line 928, in create_order
    order = self.parse_order(result, market)
  File "C:\Users\xxx\xxx\env\lib\site-packages\ccxt\aofex.py", line 707, in parse_order
    timestamp = self.parse8601(self.safe_string(order, 'ctime')) - 28800000  # 8 hours, adjust to UTC
TypeError: unsupported operand type(s) for -: 'NoneType' and 'int'
```

**Comment by kroitor** [comment]
@zaneran9 

1. You don't have the rate-limiter enabled: 

- https://github.com/ccxt/ccxt/wiki/Manual#rate-limit

2. Can you paste your verbose request/response? Plz read this: https://github.com/ccxt/ccxt/wiki/FAQ#what-is-required-to-get-help

```Python
import ccxt

symbols = 'BV/AQ'
amount = 1
price = 2.0
aofex = ccxt.aofex({'enableRateLimit': True})
aofex.apiKey = 'xxxxx'
aofex.secret = 'xxxxx'

aofex.load_markets()
aofex.verbose = True
a = aofex.create_limit_buy_order(symbols, amount, price)
print(a)
```

↑ Paste your request+response here plz.

**Comment by kroitor** [comment]
This is fixed in CCXT version 1.27.49. Let us know if you still have issues with it after upgrading to the most recent version of CCXT. Thank you!

**Comment by zaneran9** [comment]
It works, Thanks!
